content/docs/command-reference/run.md

```diff
@@ -150,12 +150,12 @@ data pipeline (e.g. random numbers, time functions, hardware dependency, etc.)
   executing the `command`.
 
 - `--no-exec` - create a stage file, but do not execute the `command` defined in
-  it, nor track dependencies or outputs with DVC. In the DVC-file contents, the
-  file hash values will be empty; They will be populated the next time this
-  stage is actually executed. DVC will also add your outputs to `.gitignore`,
-  same as it would do without `--no-exec`. This is useful if, for example, you
-  need to build a pipeline (dependency graph) first, and then run it all at
-  once.
+  it, nor commit dependencies or outputs to DVC. In the DVC-file contents, the
+  file hash values will be empty (like with `--no-commit`, explained below);
+  They will be populated the next time this stage is actually executed or with
+  `dvc commit`. DVC will also add your outputs to `.gitignore`, same as it would
+  do without `--no-exec`. This is useful if, for example, you need to build a
+  pipeline (dependency graph) first, and then run it all at once.
```

Comment:
This is per https://discord.com/channels/485586884165107732/485596304961962003/715285386053550140

Is it correct?

Reply:
This is good!

`commit dependencies or outputs` - we don't use `commit` term in this sense anywhere else - better to say "save" or "cache" ... or be more explicit if we don't have a good term

Reply:
Updated in 980c1fb9